README.md

```diff
@@ -62,13 +48,13 @@ View more [BUX documentation](https://getbux.io).
 
 <br />
 
-### Connection as a BUX admin
+### Connection as an SPV Wallet admin
 
-You can add the admin key to the BUX client after initialisation to support admin requests (IE: registering an xPub).
+You can add the admin key to the SPV Wallet client after initialisation to support admin requests (IE: registering an xPub).
 
 ```javascript
 const adminKey = 'xprv.....';
-buxClient.SetAdminKey(adminKey);
+spvWalletClient.SetAdminKey(adminKey);
```

Comment:
IMHO as a variable name, we can use just a `client` in most of the code.
As mostly the name of variable is in some context, for example in context of creation, like
```
const client = new SpvWalletClient(...)
```

Reply:
Changed